sw/device/lib/dif/dif_clkmgr.h

```diff
@@ -40,6 +40,99 @@ typedef uint32_t dif_clkmgr_gateable_clock_t;
  */
 typedef uint32_t dif_clkmgr_hintable_clock_t;
 
+typedef enum dif_clkmgr_measure_clock {
+  /**
+   * The Io clock.
+   */
+  kDifClkmgrMeasureClockIo,
+  /**
+   * The Io_div2 clock.
+   */
+  kDifClkmgrMeasureClockIoDiv2,
+  /**
+   * The Io div4 clock.
+   */
+  kDifClkmgrMeasureClockIoDiv4,
+  /**
+   * The Main clock.
+   */
+  kDifClkmgrMeasureClockMain,
+  /**
+   * The Usb clock.
+   */
+  kDifClkmgrMeasureClockUsb,
+} dif_clkmgr_measure_clock_t;
+
+typedef enum dif_clkmgr_recov_err_type {
+  /**
+   * A recoverable measurement error for IO clock.
+   */
+  kDifClkmgrRecovErrTypeIoMeas = (1u << 0),
```

Comment:
You can drop these parens.

Reply:
Certainly... I think they I added them after dif_pwrmgr.h. Done.